internal/babysitter/babysitter.go

```diff
@@ -158,8 +158,7 @@ func (b *Babysitter) SelfAddr() string { return b.selfAddr }
 // exportMetrics periodically exports metrics.
 func (b *Babysitter) exportMetrics() {
 	// Time interval at which metrics are exported.
-	const metricExportInterval = 15 * time.Second
-	ticker := time.NewTicker(metricExportInterval)
+	ticker := time.NewTicker(b.cfg.Telemetry.Metrics.ExportInterval.AsDuration())
```

Comment:
What happens if the `ExportInterval` is zero? Can we explicitly check for this?

Reply:
Good point. I did the validation in `internal/tool/deploy` when we build the options; we enforce `export_interval` to be at least 10 seconds, wdyt?

Reply:
SGTM